librelane/steps/step.py

```diff
@@ -13,73 +13,73 @@
 # limitations under the License.
 from __future__ import annotations
 
-import os
```

Comment:
Our import orders are currently "length then alphabetically" for all source files. It's not ideal but changing it for one file is not very consistent either.